crates/daphne/src/vdaf/prio3_latest.rs

```diff
@@ -0,0 +1,500 @@
+// Copyright (c) 2022 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+//! Parameters for the [Prio3 VDAF](https://datatracker.ietf.org/doc/draft-patton-cfrg-vdaf/).
+
+use crate::{
+    fatal_error,
+    messages::TaskId,
+    vdaf::{VdafError, VdafVerifyKey},
+    DapAggregateResult, DapMeasurement, Prio3Config, VdafAggregateShare, VdafPrepShare,
+    VdafPrepState,
+};
+
+use super::{prep_finish, prep_finish_from_shares, shard_then_encode, unshard};
+
+use prio::{
+    codec::ParameterizedDecode,
+    flp::Type,
+    vdaf::{
+        prio3::{Prio3, Prio3InputShare, Prio3PrepareShare, Prio3PrepareState, Prio3PublicShare},
+        xof::Xof,
+        Aggregator,
+    },
+};
+
+const ERR_FIELD_TYPE: &str = "unexpected field type for step or message";
```

Comment:
Let's just inline this since we didn't end up using string constants for errors in this file.